### ISSUE TITLE
mix google_apis.fetch fails because Google Webstore endpoint 404s

### ISSUE THREAD

**Issue by iamwilhelm** [GoogleCloudPlatform/elixir-google-api#9]
when I run mix `google_apis.fetch`, I get an error.
```
 > mix google_apis.fetch
18:16:23.936 [info]  FOUND: https://acceleratedmobilepageurl.googleapis.com/$discovery/rest?version=v1
18:16:24.091 [info]  FOUND: https://www.googleapis.com/discovery/v1/apis/adexchangebuyer/v1.4/rest
18:16:24.279 [info]  FOUND: https://adexchangebuyer.googleapis.com/$discovery/GOOGLE_REST_SIMPLE_URI?version=v2beta1
[clipped]
18:16:45.640 [info]  FOUND: https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
** (MatchError) no match of right hand side value: {:error, "Error received status: 404 from discovery endpoint"}
    lib/google_apis/discovery.ex:23: GoogleApis.Discovery.fetch/1
    lib/google_apis.ex:54: GoogleApis.fetch/1
    (elixir) lib/enum.ex:675: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:675: Enum.each/2
    (mix) lib/mix/task.ex:301: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:75: Mix.CLI.run_task/2
```

I don't think it's calendar, but the webstore api, that's missing. hitting this url confirms: `https://www.googleapis.com/discovery/v1/apis/chromewebstore/v1.1/rest`

**Comment by iamwilhelm** [comment]
Other endpoints that failed:

- https://www.googleapis.com/discovery/v1/apis/fitness_internal/v0dogfood/rest
- https://www.googleapis.com/discovery/v1/apis/reportcard/v1/rest
- https://www.googleapis.com/discovery/v1/apis/devprojects/v1/rest
- https://www.googleapis.com/discovery/v1/apis/chromewebstore/v1.1/rest
- https://www.googleapis.com/discovery/v1/apis/cih/v3/rest

When `mix google_apis.discover` is called, I believe it writes `apis-candidate.json` instead of `apis.json`. Hence, the old api specs don't get replaced, and the mix tasks read from the old list.

I've tried to change the location of the file read in `lib/google_apis/api_config.ex` in `load_all()`, but that resulted in the following error:

```
> mix google_apis.fetch
Compiling 1 file (.ex)
Fetching ....
** (FunctionClauseError) no function clause matching in Regex.run/3    
    
    The following arguments were given to Regex.run/3:
    
        # 1
        ~r/(https:\/\/.*\.googleapis.com\/\$discovery\/)([^?]*)(\?.*)?/
    
        # 2
        nil
    
        # 3
        []
    
    Attempted function clauses (showing 1 out of 1):
    
        def run(%Regex{re_pattern: compiled}, string, options) when is_binary(string)
    
    (elixir) lib/regex.ex:277: Regex.run/3
    lib/google_apis/discovery.ex:20: GoogleApis.Discovery.fetch/1
    lib/google_apis.ex:54: GoogleApis.fetch/1
    (elixir) lib/enum.ex:675: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:675: Enum.each/2
    (mix) lib/mix/task.ex:301: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:75: Mix.CLI.run_task/2
```
This is weird, because the formats for each is the same. It seems like it got an url that was blank, and hence the error.

I think it's probably easily fixable, but I just removed the offending endpoints in `apis.json`, and ran `mix google_apis.fetch` to be on my merry way for the time being.

**Comment by iamwilhelm** [comment]
I fixed these issues in PR#10

**Comment by chingor13** [comment]
Thanks for the report. Those 5 APIs should not be included and I'll remove them.
